charts/stable/anonaddy/templates/_secrets.tpl

```diff
@@ -0,0 +1,16 @@
+{{/* Define the secrets */}}
+{{- define "anonaddy.secrets" -}}
+{{- $secretName := (printf "%s-anonaddy-secrets" (include "tc.v1.common.lib.chart.names.fullname" $)) }}
+
+{{- $appKey := randAlphaNum 64 -}}
+{{- $secretKey := randAlphaNum 64 -}}
+
+ {{- with lookup "v1" "Secret" .Release.Namespace $secretName -}}
+   {{- $authKey = index .data "APP_KEY" | b64dec -}}
```

Comment:
apparently it should be b64 encoded
https://github.com/anonaddy/docker/blob/master/README.md?plain=1#L107